app/course/controllers/CourseCtrl.js

```diff
@@ -274,6 +274,25 @@ courseApp.controller('CourseCtrl', ['$scope', '$rootScope', '$routeParams', '$ti
 
 		$scope.updateSection = function (section) {
 			courseActionCreators.updateSection(section);
+
+			if (isNumber(section.sequenceNumber) == false) {
+				return;
+			}
+
+			var sectionGroup = $scope.view.state.sectionGroups.list[section.sectionGroupId];
```

Comment:
Add a comment as to what everything below the guard clause is doing, e.g. the special number-sequence behavior.